platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -452,6 +452,7 @@ function _getViewportComponent(
     }
   }
 
+  console.log("Can't show displaySet", SOPClassHandlerId, displaySets[0]);
```

Comment:
This is useful for debugging - we shouldn't hit this unless things have gone wrong.